index.js

```diff
@@ -3656,6 +3656,42 @@
       [Fn(a, b), m(a), m(a)],
       Z.sortBy);
 
+  //# zipWith :: (Applicative m, Foldable m, Monoid m) => (a -> b -> c) -> m a -> m b -> Arrray c
```

Comment:
There are two types which seem sensible to me:

  - `(a -> b -> c) -> Array a -> Array b -> Array c`
  - `(a -> b -> c) -> m a -> m b -> m c` (with appropriate constraints on `m`)

Supporting various input types but fixing the output type does not feel right to me.

Reply:
Yes! How can we do the later though? I tried to use S.type wth S.of but the type class in S.type is a string. Any tip?